GetIntoTeachingApi/AppStart/StartupStaging.cs

```diff
@@ -0,0 +1,52 @@
+﻿using GetIntoTeachingApi.AppStart.Hangfire;
+using GetIntoTeachingApi.AppStart.Prometheus;
+using Hangfire;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace GetIntoTeachingApi.AppStart
+{
+    public class StartupStaging : Startup
```

Comment:
Is there much of a difference between staging and production? worth having staging inherit from production and make any further tweaks in the subclass if they are very similar?